pkg/wheel/parse.go

```diff
@@ -0,0 +1,29 @@
+package wheel
+
+import (
+	"bufio"
+	"io"
+	"net/textproto"
+
+	"github.com/aquasecurity/go-dep-parser/pkg/types"
+	"golang.org/x/xerrors"
+)
+
+// Parse parses METADATA file for library Name and Version
+// currently does not parse its possible dependencies
+// https://packaging.python.org/specifications/core-metadata/#requires-dist-multiple-use
```

Comment:
should we scan for sub-dependencies? if so, I haven't found a method to pinpoint what current version is being used based on `Requires-Dist` in `METADATA` files, which I believe is necessary [here](https://github.com/aquasecurity/trivy/blob/a0cd5d70ae206ff9619c2d6d63a14830289fe520/pkg/detector/library/python/compare.go#L20)

Reply:
If I understand correctly, we don't have to extract sub-dependencies. If they are actually necessary, those wheel would exist in the same system.